readme.md

```diff
@@ -385,31 +432,32 @@ test(async t => {
 });
 ```
 
-*You don't have to manually call `t.end()`.*
-
 ### Observable support
 
 AVA comes with builtin support for [observables](https://github.com/zenparsing/es-observable).
+If you return an observable from a test, AVA will automatically consume it to completion before ending the test.
+*You don't have to use legacy `test.async` mode or `t.end()`.*
```

Comment:
FYI - I did make the change so we throw if they return a promise / observable. However, I don't think the wording here needs to be changed. It is not an error to use Observables in `test.async` tests, it's just an error to **return** an Observable.

We could add more language here fully explaining that, but I'm not sure it is necessary. The semantics of returning an async object are now well defined (the test ends when the async object ends). It doesn't make sense to combine that behavior with `t.end()`. Telling them they can't do something that doesn't make sense seems to be more noise than helpful.